### ISSUE TITLE
Misinterpretation of external links as external ressource

### ISSUE THREAD

**Issue by josci** [Kozea/CairoSVG#220]
Hi,

I'm trying to convert a quite simple svg file to PDF: 

```
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 595.276 841.89" width="100%" xml:space="preserve">
    <defs>
        <link xmlns="http://www.w3.org/1999/xhtml" />
    </defs>
    <a transform="translate(48.189,68.0315)" xlink:href="http://cnn.com">
    </a>
</svg>
```

Cairo throws an error:
```
File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\__init__.py", line 79, in svg2pdf
    output_height=output_height)
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\surface.py", line 147, in convert
    output_width, output_height)
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\surface.py", line 220, in __init__
    self.draw(tree)
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\surface.py", line 441, in draw
    self.draw(child)
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\surface.py", line 381, in draw
    TAGS[node.tag](self, node)
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\text.py", line 85, in text
    node.parent.get('{http://www.w3.org/1999/xlink}href', ''))
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\url.py", line 138, in parse_url
    url = normalize_url(url.strip('\'"'))
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\site-packages\cairosvg\url.py", line 55, in normalize_url
    url = Path(url).resolve().as_uri()
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\pathlib.py", line 1139, in resolve
    s = self._flavour.resolve(self, strict=strict)
  File "C:\ProgramData\Anaconda3\envs\lhub\lib\pathlib.py", line 193, in resolve
    s = self._ext_to_normal(_getfinalpathname(s))
OSError: [WinError 123] Die Syntax für den Dateinamen, Verzeichnisnamen oder die Datenträgerbezeichnung ist falsch: 'C:\\tmp\\http:\\cnn.com'
```

It seems like the external links is interpreted as external ressource.

I'm using CairoSVG 2.2.1 on windows 7 (32bit)

**Comment by josci** [comment]
"Die Syntax für den Dateinamen, Verzeichnisnamen oder die Datenträgerbezeichnung ist falsch"

translates to: "The syntax of the filename, directory name or the drive (hdd) name is wrong"

**Comment by elzapp** [comment]
The svg `a`-tag takes a href attribute on it's own. It doesn't use the xlink namespace for the href.

using xlink:href makes the parser try to use the target for the link instead of just linking to it. 

`<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 595.276 841.89" width="100%" xml:space="preserve">
    <defs>
        <link xmlns="http://www.w3.org/1999/xhtml" />
    </defs>
    <a transform="translate(48.189,68.0315)" href="http://cnn.com">
    </a>
</svg>`

**Comment by liZe** [comment]
It's fixed now, probably with #236.
